AudioContextInterruptedState/explainer.md

```diff
@@ -0,0 +1,212 @@
+
+## Goals
+
+The main goal of this proposal is to allow the UA to be able to interrupt an `AudioContext` playback when needed, given that there are a couple of user scenarios - e.g., incoming phone calls - and proposed web API's that could make good use of this functionality.
+
+### "media-playback-while-not-visible" permission policy
+
+The ["media-playback-while-not-visible" permission policy](https://github.com/MicrosoftEdge/MSEdgeExplainers/blob/main/IframeMediaPause/iframe_media_pausing.md) allows the UA to pause media playback of unrendered iframes. However, since the Web Audio API [does not allow the UA to suspend](https://webaudio.github.io/web-audio-api/#dom-audiocontext-suspended-by-user-slot) the audio playback, the proposed permission policy has no mechanism available to pause media playback.
+
+### Audio Session API
+
+Similarly, the [Audio Session API](https://w3c.github.io/audio-session/) could also make use of the `"interrupted"` state. Whenever the [`AudioSessionState`](https://w3c.github.io/audio-session/#enumdef-audiosessionstate) is `"interrupted"`, the AudioContext of the document would also transition to the `"interrupted"` state. As a matter of fact, the `AudioContext`'s `"interrupted"` state has been [implemented on WebKit](https://github.com/WebKit/WebKit/blob/1e8ea6e4777297ce82e6c911caa7cce2cc32e6a9/Source/WebCore/Modules/webaudio/AudioContextState.idl) and is currently being used by the Audio Session API. 
+
+In Safari, on MacOS Sonoma 14.5, if there an active web page with the `AudioContext` in the `"running"` state, and if we set the audio session type to `"auto"` with `navigator.audioSession.type = "auto"`; whenever the laptop's screen is locked, the AudioContext will transition to the `"interrupted"` state. When the screen is unlocked, the state will automatically move to `"running"` again.
+
+### Exclusive access to audio hardware
+
+There are scenarios where another application may acquire exclusive access to audio hardware. For example, when a phone call is in progress. In this situation, if there is already an `AudioContext` in the `running` state, it makes sense that UA puts into in the `interrupted` while the call is in progress.

```

Comment:
nit: "it makes sense that UA puts into in the interrupted while the call is in progress." 

Maybe update to something clearer like the following? 

"it makes sense that the UA pauses the AudioContext using the interrupted state while the call is in progress."